scheduler/scheduler-api/src/test/java/org/ow2/proactive/scheduler/common/job/factories/spi/model/factory/GlobalFileParserValidatorTest.java

```diff
@@ -0,0 +1,97 @@
+/*
+ * ProActive Parallel Suite(TM):
+ * The Open Source library for parallel and distributed
+ * Workflows & Scheduling, Orchestration, Cloud Automation
+ * and Big Data Analysis on Enterprise Grids & Clouds.
+ *
+ * Copyright (c) 2007 - 2017 ActiveEon
+ * Contact: contact@activeeon.com
+ *
+ * This library is free software: you can redistribute it and/or
+ * modify it under the terms of the GNU Affero General Public License
+ * as published by the Free Software Foundation: version 3 of
+ * the License.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ *
+ * If needed, contact us to obtain a release under GPL Version 2 or 3
+ * or a different license than the AGPL.
+ */
+package org.ow2.proactive.scheduler.common.job.factories.spi.model.factory;
+
+import static org.mockito.Mockito.when;
+import static org.ow2.proactive.scheduler.common.SchedulerConstants.GLOBALSPACE_NAME;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.ow2.proactive.scheduler.common.SchedulerSpaceInterface;
+import org.ow2.proactive.scheduler.common.exception.NotConnectedException;
+import org.ow2.proactive.scheduler.common.exception.PermissionException;
+import org.ow2.proactive.scheduler.common.job.factories.spi.model.ModelValidatorContext;
+import org.ow2.proactive.scheduler.common.job.factories.spi.model.exceptions.ConversionException;
+import org.ow2.proactive.scheduler.common.job.factories.spi.model.exceptions.ModelSyntaxException;
+import org.ow2.proactive.scheduler.common.job.factories.spi.model.exceptions.ValidationException;
+
+
+public class GlobalFileParserValidatorTest {
+    final String existGlobalFilePath = "global folder/exist-file.txt";
+
+    final String notExistGlobalFilePath = "not-exist-file.png";
+
+    @Mock
+    private ModelValidatorContext context;
+
+    @Mock
+    private SchedulerSpaceInterface schedulerSpaceInterface;
+
+    @Before
+    public void init() throws NotConnectedException, PermissionException {
+        MockitoAnnotations.initMocks(this);
+        when(context.getSpace()).thenReturn(schedulerSpaceInterface);
+        when(schedulerSpaceInterface.checkFileExists(GLOBALSPACE_NAME, existGlobalFilePath)).thenReturn(true);
+        when(schedulerSpaceInterface.checkFileExists(GLOBALSPACE_NAME, notExistGlobalFilePath)).thenReturn(false);
+    }
+
+    @Test
+    public void testValidGlobalFileOK() throws ModelSyntaxException, ValidationException, ConversionException {
```

Comment:
I think that it could improve your test naming by renaming it such as `testThatValidGlobalFileIsOk`. Have a "sentence name" make it easier to read. It doesn't bother to have long name as the test is never called. Same comment for next tests. It would also be nice to use google truth but I guess you didn't used it because it was not available in this module.

Reply:
I've improved the test function names. I think it's a good advise to use google truth, but maybe we could do it in the next release ?

Reply:
Those are minor comments that's why I approved the PR. I let you manage your priorities.